specification/search/data-plane/Microsoft.Azure.Search.Data/preview/2017-11-11-preview/searchindex.json

```diff
@@ -339,16 +397,204 @@
+            "x-ms-client-name": "OrderBy",
+            "x-ms-parameter-grouping": {
+              "name": "SuggestParameters"
+            }
+          },
+          {
+            "name":"searchFields",
+            "in": "query",
+            "type": "array",
+            "items": {
+              "type": "string"
+            },
+            "description": "The list of comma-separated field names to search for the specified search text. Target fields must be enabled for suggestions.",
+            "x-ms-parameter-grouping": {
+              "name": "SuggestParameters"
+            }
+          },
+          {
+            "name":"$select",
+            "in": "query",
+            "type": "string",
+            "description": "The comma-separated list of fields to retrieve. If unspecified, all fields marked as retrievable in the schema are included.",
+            "x-ms-client-name": "SelectStr",
+            "x-ms-parameter-grouping": {
+              "name": "SuggestParameters"
+            }
+          },
+          {
+            "name":"$top",
+            "in": "query",
+            "type": "integer",
+            "format": "int32",
+            "description": "The number of suggestions to retrieve. The value must be a number between 1 and 100. The default is to 5.",
+            "x-ms-client-name": "Top",
+            "x-ms-parameter-grouping": {
+              "name": "SuggestParameters"
+            }
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ClientRequestIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ClientAcceptParameter"
+          }
+        ],
+        "x-ms-request-id": "request-id",
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/DocumentSuggestResult"
+            }
+          }
+        }
+      },
+      "post": {
+        "tags": [
+          "Documents"
+        ],
+        "operationId": "DocumentsProxy_SuggestPost",
+        "externalDocs": {
+          "url": "https://docs.microsoft.com/rest/api/searchservice/suggestions"
+        },
+        "x-ms-examples": {
+          "SuggestIndexPostSearch": { "$ref": "./examples/SearchIndexPostSuggest.json" }
+        },
+        "description": "Suggests query terms based on input text and matching documents in the Azure Search index.",
```

Comment:
We should rephrase the description to make it clear that it's suggesting documents, not query terms. Otherwise it might be confusing to tell the difference between this and Autocomplete.

Reply:
Yes, I agree. The doc is from https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.search.idocumentsoperations.suggestwithhttpmessagesasync which we should update.

Reply:
How does this sound? "Returns matching text with the document key in the Azure Search index."

Reply:
That's a bit literal; We should try to explain the bigger picture. How about: "Suggests documents in the Azure Search index that match the given partial query text."?

The document you linked to is the SDK reference, which is generated from the C# code of the GA SDK. Once these SDK changes get merged from preview to GA later this calendar year, those docs should be automatically updated with whatever you put in the Swagger.

Reply:
Sounds good. Thanks!